core/src/main/scala/org/broadinstitute/dsde/rawls/monitor/CreatingBillingProjectMonitor.scala

```diff
@@ -99,8 +99,8 @@ trait CreatingBillingProjectMonitor extends LazyLogging {
               case util.Failure(t) =>
                 logger.info(s"Failure creating project $project", t)
                 Future.successful(project.copy(status = CreationStatuses.Error, message = Option(t.getMessage)))
-              case Success(operations) => datasource.inTransaction { dataAccess =>
-                dataAccess.rawlsBillingProjectQuery.insertOperations(operations).map(_ => project)
+              case Success(ops) => datasource.inTransaction { dataAccess =>
```

Comment:
`operations` is a value in scope already so this may be shadowing that.  Happy to make the name longer as long as it's unique.